latex2sympy.py

```diff
@@ -618,6 +629,9 @@ def convert_func(func):
         if name == "exp" or name == "exponentialE":
             expr = sympy.exp(arg)
 
+        if name in ["gcd", "lcm"]:
```

Comment:
@tirkarp this makes me wonder if we should update the other statements to match (line 629, 617). While it doesn't change the functionality, it's nice to follow the same syntax when we can.

Reply:
pushed.